docs/products/kafka/kafka-connect/howto/configure-aws-secrets-manager.md

```diff
@@ -9,12 +9,18 @@ Configure and use [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanag
 
 ## Prerequisites
 
-- Access to the [Aiven Console](https://console.aiven.io/).
 - [Aiven for Apache Kafka service with Apache Kafka Connect](/docs/products/kafka/kafka-connect/get-started)
   set up and running.
 - [Aiven CLI](/docs/tools/cli).
+- [Aiven Terraform provider](https://registry.terraform.io/providers/aiven/aiven/latest/docs)
+  installed.
 - [AWS Secrets Manager access key and secret key](https://docs.aws.amazon.com/secretsmanager/latest/userguide/auth-and-access.html).
 
+:::note
+The integration with AWS Secrets Manager is not yet available on the Aiven Console.
+This feature can only be configured using the Aiven API, Aiven CLI, or Terraform.
+:::
+
```

Comment:
I would advise against this kind of docs, it's more maintenance later if this ever becomes a Console feature, and it creates a pattern users might expect everywhere: tell where things are _not_ available vs only telling them where things are available.

Ideally, it should be as simple as "We don't have a Console tab in the page, so you can't do it with the Console".

Also this is a repetition, we already say where the feature is available below, no need to say it twice :)

Reply:
I see your point, but I find the comment a bit mixed. On one hand, you’re suggesting we avoid stating where features are not available to prevent user expectations. On the other hand, mentioning the absence of a Console tab seems to contradict that approach, which is where I’m a bit confused.

Currently, I’ve shortened the statement to simply mention that this integration is not available via the Console.